astropy/table/jsviewer.py

```diff
@@ -23,7 +23,7 @@ class Conf(_config.ConfigNamespace):
         'The URL to the jquery datatables library.')
 
     css_urls = _config.ConfigItem(
-        ['https://cdn.datatables.net/1.10.9/css/jquery.dataTables.min.css'],
+        ['https://cdn.datatables.net/1.10.9/css/jquery.dataTables.css'],
```

Comment:
For the CSS, I think @saimn noted above that the minified version is not much smaller than the regular one so there seems little point in having the minified version at all. The source, though, should be the mins (which are quite a bit smaller/more optimized)